src/services/email/email.service.ts

```diff
@@ -0,0 +1,23 @@
+import * as SendGridMail from '@sendgrid/mail';
+import * as dotenv from 'dotenv';
+
+import { EmailInfo } from './email.definition';
+
+dotenv.config();
+SendGridMail.setApiKey(process.env.SENDGRID_API_KEY);
+const senderEmail: string = process.env.SENDER_EMAIL_ADDRESS;
+
+/*
+ * Send one or more emails to one or more respective recipients
+ * for each of those emails
+ *
+ * TODO: Add logging when there is a logging set up
+ */
+const _sendEmails = async (emailInfoArr: Array<EmailInfo>): Promise<void> => {
+  const emailsToSend = emailInfoArr.map(emailInfo => {
+    emailInfo.from = senderEmail;
```

Comment:
hm from is still required so do callers leave it empty?

Reply:
Oh shoot you're right. Uhhhh if callers leave it empty it's gonna give an error in the text editor, I just tested it.

Reply:
I think I might have a solution to this tho.

Reply:
It’s ok - I’d rather we setup error handling and logging soon, both of which would be rather straightforward 
We can discuss this tmrw

Reply:
I'm also working on the error handling middleware. I'll make a PR as soon as I think I'm done with it.